lib/sle2docker/prebuilt_image.rb

```diff
@@ -103,7 +103,7 @@ def compute_repository_and_tag
       match = regexp.match(@image_name)
       if match.nil?
         fail DockerTagError,
-             "Cannot calculate the Docker tag for #{@image_name}"
+             "Docker image #{@image_name} not found"
```

Comment:
This is not really the case. Bear in mind that this comes after parsing through a regexp. So it very well be that the image exists but it doesn't follow the expected format. So, I propose this on failure:

- Check if the image exist in docker.
- If it exists, then write a message that it doesn't follow the proper format (while also writing the format to be followed and an example). If it doesn't exist, then print this "image not found" message.

Does it make sense ?